### ISSUE TITLE
Add a config for ignoring warning about class naming case warning

### ISSUE THREAD

**Issue by tksvn-vanna** [TKSVN/constant_keys_generator#3]
## Problem

In content of generated file, some warnings about class naming case are appeared

> The type name '_LocaleKeys_Common_M' isn't an UpperCamelCase identifier.
Try changing the name to follow the UpperCamelCase style.

## Solution

Add the following line to ignore all warning about naming case

```dart
// ignore_for_file: camel_case_types
```

**Comment by Vishnu-0609** [comment]
"_LocaleKeys_Common_M" this content not found in your project

**Comment by tksvn-vanna** [comment]
> "_LocaleKeys_Common_M" this content not found in your project

@Vishnu-0609 
`'_LocaleKeys_Common_M'` is not contained in our source.
It is located in a generated dart file with name `locale_keys.generated.dart`.
As you know `_LocaleKeys_Common_M` is not a valid name with dart/flutter naming convention for class, but we have to do that to make all classes of generated file are private.

The following sample code will make you more understand about what is we actually do

Original json file
```json
{
    "common": {
        "L": {
            "toastErrorTitle": "Đã có lỗi xảy ra",
            "osVersionNotSupported": "Thiết bị/OSVersion này không được hỗ trợ",
            "cancel": "Huỷ bỏ",
            "notification": "Thông báo",
            "settingChange": "Thay đổi cài đặt",
            "completion": "Hoàn thành",
            "delete": "Xoá",
            "back": "Quay lại",
            "next": "Tiếp theo"
        },
        "M": {
            "tokenExpired": "Phiên làm việc đã hết hạn, vui lòng đăng nhập lại.",
            "unexpectedError": "Đã có lỗi xảy ra, xin vui lòng thử lại hoặc liên lạc với quản trị hệ thống.",
            "noData": "Không có dữ liệu.",
            "changePhotoOnSettingScreen": "Vui lòng thay đổi quyền ảnh trên màn hình cài đặt.",
            "cantAccessCamera": "Không thể truy cập vào máy ảnh."
        }
    }
}
```

Generated dart file
```dart
class _LocaleKeys_Common_L {
  final String toastErrorTitle = 'common.L.toastErrorTitle';
  final String osVersionNotSupported = 'common.L.osVersionNotSupported';
  final String cancel = 'common.L.cancel';
  final String notification = 'common.L.notification';
  final String settingChange = 'common.L.settingChange';
  final String completion = 'common.L.completion';
  final String delete = 'common.L.delete';
  final String back = 'common.L.back';
  final String next = 'common.L.next';
  _LocaleKeys_Common_L();
}


class _LocaleKeys_Common_M {
  final String tokenExpired = 'common.M.tokenExpired';
  final String unexpectedError = 'common.M.unexpectedError';
  final String noData = 'common.M.noData';
  final String changePhotoOnSettingScreen = 'common.M.changePhotoOnSettingScreen';
  final String cantAccessCamera = 'common.M.cantAccessCamera';
  _LocaleKeys_Common_M();
}


class _LocaleKeys_Common {
  final _LocaleKeys_Common_L L = _LocaleKeys_Common_L();
  final _LocaleKeys_Common_M M = _LocaleKeys_Common_M();
  _LocaleKeys_Common();
}
```

**Comment by tksvn-vanna** [comment]
I've resolved with release of v1.0, ref PR #8 for more detail.
